dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/RenormalizationProcessor.java

```diff
@@ -115,37 +126,101 @@ private Variant renormalize(IVariant variant) {
-                i.getSamplesData()
+                variantSourceEntry.getFileId(),
+                variantSourceEntry.getStudyId(),
+                variantSourceEntry.getSecondaryAlternates(),
+                variantSourceEntry.getFormat(),
+                renormalizeStatistics(variantSourceEntry, renormalizedReference, renormalizedAlternate),
+                renormalizeAttributes(variantSourceEntry.getAttributes(), variant, renormalizedReference,
+                                      renormalizedAlternate),
+                variantSourceEntry.getSamplesData()
         );
     }
 
+    private Map<String, VariantStatistics> renormalizeStatistics(IVariantSourceEntry variantSourceEntry,
+                                                                 String renormalizedReference,
+                                                                 String renormalizedAlternate) {
+        Map<String, VariantStatistics> renormalizedStatistics = new HashMap<>();
+        for (Map.Entry<String, VariantStatistics> statisticsEntry : variantSourceEntry.getCohortStats().entrySet()) {
+            VariantStatistics stats = statisticsEntry.getValue();
+            String renormalizedMafAllele = renormalizeMafAllele(stats, renormalizedReference, renormalizedAlternate);
+            VariantStatistics variantStatistics = new VariantStatistics(renormalizedReference,
+                                                                        renormalizedAlternate,
+                                                                        stats.getVariantType(),
+                                                                        stats.getMaf(),
+                                                                        stats.getMgf(),
+                                                                        renormalizedMafAllele,
+                                                                        stats.getMgfGenotype(),
+                                                                        stats.getMissingAlleles(),
+                                                                        stats.getMissingGenotypes(),
+                                                                        stats.getMendelianErrors(),
+                                                                        stats.getCasesPercentDominant(),
+                                                                        stats.getControlsPercentDominant(),
+                                                                        stats.getCasesPercentRecessive(),
+                                                                        stats.getControlsPercentRecessive());
+            variantStatistics.setRefAlleleCount(stats.getRefAlleleCount());
+            variantStatistics.setAltAlleleCount(stats.getAltAlleleCount());
+            variantStatistics.setRefAlleleFreq(stats.getRefAlleleFreq());
+            variantStatistics.setAltAlleleFreq(stats.getAltAlleleFreq());
```

Comment:
These 4 variables (and many others really) can't be obtained from dbSNP so the calls to the setters can be dropped.